packages/cozy-procedures/src/components/PersonalDataFormContainer.jsx

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+import get from 'lodash/get'
+import context from '../redux/context'
+import PersonalDataForm from './PersonalDataForm'
+import { update as updateFormData } from '../redux/personalDataSlice'
+
+const mapStateToProps = state => ({
+  formData: get(state, 'personalData')
+})
+
+const mapDispatchToProps = dispatch => ({
+  updateFormData: value => dispatch(updateFormData(value))
+})
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps,
+  null,
+  { context }
```

Comment:
So that the redux store for `cozy-procedures` is isolated from the app's store. It was based on [this](https://react-redux.js.org/using-react-redux/accessing-store#multiple-stores) but… that might be for another redux version.

Not sure if there are other "standard" ways to isolate a redux store as part of a bigger app?